packages/server/events-processing-platform/domain/opportunity/aggregate/aggregate.go

```diff
@@ -68,3 +70,21 @@ func (a *OpportunityAggregate) onOpportunityCreate(evt eventstore.Event) error {
 
 	return nil
 }
+
+func (a *OpportunityAggregate) onOpportunityCreateRenewal(evt eventstore.Event) error {
+	var eventData event.OpportunityCreateRenewalEvent
+	if err := evt.GetJsonData(&eventData); err != nil {
+		return errors.Wrap(err, "GetJsonData")
+	}
+
+	a.Opportunity.ID = a.ID
+	a.Opportunity.Tenant = a.Tenant
+	a.Opportunity.OrganizationId = eventData.ContractId
```

Comment:
shouldn't that be = `eventData.OrganizationId` ?

Reply:
correct, fixed